### PR TITLE
Add support to ELB trigger by

### PR DESCRIPTION
**Pull request by doriaviram**

### CODE REVIEW

File: src/test/unit/parsers/test_utils.py [hunk 1]
```diff
@@ -239,6 +239,15 @@ def test_recursive_json_join(d1, d2, result):
                 "detailType": "Scheduled Event",
             },
         ),
+        (  # elb example trigger
+            {
+                "httpMethod": "GET",
+                "resource": "resource",
+                "headers": {"Host": "www.google.com"},
+                "requestContext": {"elb": "Elb", "stage": "1", "requestId": "123"},
```
Comment (efimk-lu) [inline src/test/unit/parsers/test_utils.py]: That's the way it looks like, take a real example

File: src/test/unit/parsers/test_utils.py [hunk 2]
```diff
@@ -239,6 +239,15 @@ def test_recursive_json_join(d1, d2, result):
...
+            },
+            {"triggeredBy": "unknown"},
```
Comment (efimk-lu) [inline src/test/unit/parsers/test_utils.py]: What happens when we have trigger by unknown?

File: src/test/unit/parsers/test_utils.py (hunk 1 above)
Comment (doriaviram) [reply src/test/unit/parsers/test_utils.py]: 👍

File: src/test/unit/parsers/test_utils.py (hunk 2 above)
Comment (doriaviram) [reply src/test/unit/parsers/test_utils.py]: image

**Pull request merged**